web_environment_ribbon/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2015 Francesco OpenCode Apruzzese <cescoap@gmail.com>
```

Comment:
Remove Copyright from all `__init__` file

Reply:
I removed the copyright from the __init__.py files. Can you reconsider your review, please?